src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -77,6 +79,47 @@ class OrchestrationSessionsController(private val visitSchedulerService: VisitSc
   ): List<VisitSessionDto>? =
     visitSchedulerService.getVisitSessions(prisonCode, prisonerId, min, max)
 
+  @PreAuthorize("hasRole('ROLE_ORCHESTRATION_SERVICE__PVB')")
```

Comment:
will wait to merge till we get consensus on Role and Rolename.

Reply:
updated role to VISIT_SCHEDULER. wlil update all roles to VSIP_ORCHESTRATION_SERVICE as part of a different ticket